python/cudf_polars/cudf_polars/experimental/parallel.py

```diff
@@ -225,9 +226,11 @@ def _(
     # Generate pointwise (embarrassingly-parallel) tasks by default
     child_names = [get_key_name(c) for c in ir.children]
     bcast_child = [partition_info[c].count == 1 for c in ir.children]
+
+    traced = do_evaluate_with_tracing(ir.do_evaluate, name=type(ir).__name__)
```

Comment:
(If you go with my suggested change to `do_evaluate_with_tracing`)

```suggestion
    traced = do_evaluate_with_tracing(ir)
```